### ISSUE TITLE
Collapsed comment: improve mobile support and accessibility

### ISSUE THREAD

**Issue by TomaszGasior** [TomaszGasior/RadioLista-v3#248]
Content of "comment" column in the radio table is shown directly inside table cell by default. However, the user is able to customize his radio table so  "i" icon in the circle is shown instead of the full contents. The user viewing the radio table needs to hover over that icon using his mouse to actually see the comment of specific row. 

This is implemented using "title" HTML attribute. 

What needs to be fixed:

- This implementation is not compatible with mobile devices. There's no mouse so there's no way to hover over the icon.
- This implementation is not accessible for users who don't use mouse and stick with keyboard-only navigation. There's no way to trigger the "title" there.
- Make sure new implementation is compatible with screen readers too.

---

Replaces https://github.com/TomaszGasior/RadioLista-v3/issues/107
Extra info from the end user https://github.com/TomaszGasior/RadioLista-v3/discussions/242
